src/Translation_unit.ml

```diff
@@ -138,3 +138,5 @@ let parse_print (XUnit xunit) (conf: Conf.t) iname ifile ic ofile =
   | Syntaxerr.Error _ as exc ->
       Location.report_exception Caml.Format.err_formatter exc ;
       Caml.exit 1
+  | Formatting_disabled ->
+      Stdio.Out_channel.output_string Stdio.stdout source
```

Comment:
IIUC, this will work only when outputting to stdout, but not if an output file is specified, or if formatting in-place.  A match analogous to the one on lines 123-9 above is probably needed, so that nothing is done in the `Inplace` case, and the file is copied in the `In_out` case.

And minor style nit: I would put this new match case before the others as it is an expected successful termination case, while the others are fatal errors.

Reply:
👍